app/controllers/publishers_controller.rb

```diff
@@ -129,6 +130,40 @@ def verify
   # Shown after verification is completed to encourage users to submit
   # payment information.
   def verification_done
+    @publisher = current_publisher
+    @publisher.uphold_state_token = SecureRandom.hex(64)
+    @publisher.save!
+  end
+
+  def uphold_verified
```

Comment:
Minor: Running in dev without HTTPS, when I get redirect here after Uphold oauth it's HTTP.